### DISCUSSION TITLE
Any plans to support images as input/output for gpt-4?

### DISCUSSION THREAD

**Discussion by aadiby** [neary-ai/neary#6] [General]
As the title suggest, are you (or would you) implement support for images?

**Comment by mightworks** [comment]
Hi there, support for gpt-4-vision has already been implemented. I just need to update the docs.

![vision](https://github.com/neary-ai/neary/assets/42520241/89e3e7c1-2f09-4aed-bbc0-e6b4e5d7d174)

Just select the "Vision" preset, and then you can click the image icon in the toolbar to select which image you want to send with your message. You can also select the gpt-4-vision model in any conversation to enable image input.

Let me know if you run into any issues!
